Resources/Prototypes/DeltaV/Entities/Objects/Devices/pda.yml

```
@@ -40,6 +40,123 @@
     scanningEndSound:
       path: "/Audio/Items/Medical/healthscanner.ogg"
 
+- type: entity
+  parent: BasePDA
+  id: CJPDA
```

who is CJ? this doesn't follow naming convention.
```suggestion
  id: ChiefJusticePDA
```